scripts/manage_recordings.py

```diff
@@ -0,0 +1,141 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import pathlib
+import shlex
+import sys
+
+try:
+    from dotenv import load_dotenv
+    load_dotenv()
+except:
+    pass
+
+import subprocess
+
+
+# This file contains a script for managing test recordings in the azure-sdk-assets repository with Docker.
+#
+# INSTRUCTIONS FOR USE:
+#
+# - Set GIT_TOKEN, GIT_COMMIT_OWNER, and GIT_COMMIT_EMAIL environment variables to authenticate git requests.
+#   These can be set in-process or added to a .env file at the root of or directory above your local copy of the
+#   azure-sdk-for-python repository.
+# - Set your working directory to be inside your local copy of the azure-sdk-for-python repository.
+# - Run the following command:
+#
+#     `python {path to script}/manage_recordings.py {verb} {relative path to package's assets.json file}`
+#
+#   For example, with the root of the azure-sdk-for-python repo as the working directory, you can push modified
+#   azure-keyvault-keys recordings to the assets repo with:
+#
+#     `python scripts/manage_recordings.py push sdk/keyvault/azure-keyvault-keys/assets.json`
+#
+# - In addition to "push", you can also use the "restore" or "reset" verbs in the same command format.
+#
+# For more information about how recording asset synchronization, please refer to
+# https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/documentation/asset-sync/README.md.
+
+ # Load environment variables from user's .env file
+
+CONTAINER_NAME = "azsdkengsys.azurecr.io/engsys/testproxy-lin"
```

Comment:
If they just target `azsdkengsys.azurecr.io/engsys/test-proxy` it'll just be the appropriate install for their platform 👍 

Try it!